README.md

```diff
@@ -1,194 +1,239 @@
+  * [`static/`](#static)
+  * [`tasks/`](#tasks)
+  * [`utils/`](#utils)
+* [Getting Started](#getting-started)
+  * [Base Installation](#base-installation)
+    * [Setup: Pyenv and Pipenv](#setup-pyenv-and-pipenv)
+    * [Setup: Nix](#setup-nix)
+      * [Nix: Dependency Updates](#nix-dependency-updates)
+      * [Nix: Disabling Nix](#nix-disabling-nix)
+  * [Unsupported Setup](#unsupported-setup)
+  * [Troubleshooting](#troubleshooting)
+* [Running Tests](#running-tests)
+  * [Unit Tests](#unit-tests)
+  * [Load Tests](#load-tests)
+    * [Load Tests: Local Locust Setup](#load-tests-local-locust-setup)
+      * [Setup to Run Locust Against A Local MyMove Server](#setup-to-run-locust-against-a-local-mymove-server)
+        * [Local Server Data](#local-server-data)
+      * [Setup to Run Locust Against The Load Test Env MyMove Server](#setup-to-run-locust-against-the-load-test-env-mymove-server)
+    * [Load Tests: Running Locust Locally](#load-tests-running-locust-locally)
+      * [Running Locust Command](#running-locust-command)
+        * [Workflows](#workflows)
+      * [Running Preset Load Tests](#running-preset-load-tests)
+    * [Load Tests: Running Locust from AWS](#load-tests-running-locust-from-aws)
+      * [Troubleshooting: Running Locust from AWS](#troubleshooting-running-locust-from-aws)
+    * [Load Tests: Load Test Environment](#load-tests-load-test-environment)
       * [Metrics](#metrics)
-   * [References](#references)
-
-<!-- Added by: pearl, at: Tue Dec  7 10:03:58 EST 2021 -->
-
-<!--te-->
-<!-- markdownlint-restore -->
+      * [Handling Rate Limiting](#handling-rate-limiting)
+* [Working With Load Tests](#working-with-load-tests)
+  * [Creating A New Locustfile](#creating-a-new-locustfile)
+  * [Creating TaskSets](#creating-tasksets)
+  * [Adding tasks to existing load tests](#adding-tasks-to-existing-load-tests)
+* [AWS Deployed Environment Setup](#aws-deployed-environment-setup)
+  * [Deploying to the Load Testing Environment](#deploying-to-the-load-testing-environment)
+  * [Resetting the DB After a Load Test](#resetting-the-db-after-a-load-test)
+  * [Deploying New Tests](#deploying-new-tests)
+* [Fake Data Generation](#fake-data-generation)
+  * [Creating a custom parser](#creating-a-custom-parser)
+* [References](#references)
+
+<!-- Regenerate with "pre-commit run -a markdown-toc" -->
+
+<!-- tocstop -->
 
 ## Overview
 
-MilMove is a system to help service members (and other authorized personnel) move their gear and possessions from one
-place to another. This codebase has been written to perform load tests on the MilMove app for the purpose of gathering
+MilMove is a system to help service members (and other authorized personnel) move their gear and
+possessions from one place to another.
+
+This codebase has been written to perform load tests on the MilMove app for the purpose of gathering
 data about responses times, finding breakpoints, and assessing the overall health of the system.
```

Comment:
It might be nice to say that we are only really testing the APIs and backend of the system.  It's not giving us insight into anything browser or client side.

Reply:
Ah good point, I'll add that.